### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by Adholah96**
### In this Project, I completed the following:

- [ ] Created a new directory and initialize it with git.

- [ ] Used ES6 syntax.

- [ ] Divided the code into modules.

- [ ] Kept all functionalities of the app without errors.

- [ ] Refactor functions to arrow functions.

- [ ] Used let and const in the correct way.

- [ ] Display the current date below the navigation bar.

### CODE REVIEW

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Adholah96, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linter are passing
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

I had to include some comments here because I can't access the required file for inline comments.
[REQUIRED] When I run `npm install` I get an error. I think this is because there is no `package.json` file in. your project. Kindly fix this so that I can see if your app is still working as expected.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/MainClass.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      const btnId = e.target.dataset.id;
+      Library.removeLibraryArray(btnId);
+    });
+  }
+
+  static removeLibraryArray = (id) => {
+    const libraryContainer = libraryContainer.filter((item) => item.id !== +id);
+    Library.addLocalStorage(libraryContainer);
+  }
+}
```
Comment (thecodechaser) [inline modules/MainClass.js]: - [x] When I'm trying to delete the added books, It gets deleted from the page but after refreshing the deleted books is still displayed on the page. It seems deleting the books not updating the local storage. Please make sure to fix this issue.
 - When I click on the remove button, On the console I can see this error.
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Adholah96👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Highly organized code. ✔️
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @Adholah96,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**